src/main/java/core/basesyntax/DescriptionFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DescriptionFigure {
```

Comment:
```suggestion
public interface Drawable {
```
Never use the implementation name (Figure) in the abstraction name. Abstraction does not know anything about its implementations.